packages/builder/src/components/design/PropertiesPanel/PropertyControls/FilterEditor/LuceneFilterBuilder.svelte

```diff
@@ -195,7 +195,7 @@
         />
       {:else if expression.type === "boolean"}
         <Combobox
-          disabled
+          disabled={expression.noValue}
           options={[
             { label: "True", value: true },
             { label: "False", value: false },
```

Comment:
Was doing some testing there and it looks like lucene expects strings rather than real booleans here - so these options for the combobox just also need their values updated to be wrapped in quotes to make them strings. e.g. just
```
{ label: "True", value: "true" },
{ label: "False", value: "false" },
```
My bad though, just a typo that it was hard coded disabled.

Reply:
Good catch! It did seem to work in lucene though when I tested it? Is it maybe parsing the bools on the way in?

Reply:
Weird, it definitely didn't work for me unless I made those options strings - otherwise it just shows all records. If you change it to true does it definitely filter out the false values when they are bools?

Reply:
In the same way using `{{ literal true }}` as a binding didn't work (like I would have expected it to) but string `"true"` does!

Reply:
You might not be able to set them to true in the backend UI though - I've just raised a PR to fix that
https://github.com/Budibase/budibase/pull/1571.

Reply:
I think that might have been it - it worked for falsy values when I flipped equals/not equals. Nice one on that fix 👍